### PR TITLE
CALM member addition

### PR DESCRIPTION
**Pull request by tomgron**

### CODE REVIEW

**Comment by tomgron** [comment]
@vanbroup could you review this please?

**Comment by vanbroup** [comment]
Thanks for the PR, I will review hopefully next week, else the week after.

**Review by vanbroup — CHANGES_REQUESTED** [review]

File: data/members/calm.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+id: calm
+name: CALM Oy
```
[the remark this answers is not in the document]
Comment (tomgron) [reply data/members/calm.yaml]: Change applied

File: data/members/calm.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+slogan: "Providing high quality and secure solutions for organizations around the world"
+website: https://www.calm.fi/
+
+# Short description, longer content can be placed in `content/members/`
+description: CALM Oy is provider of consulting services and producer of CLM Certificate Lifecycle Management SaaS -service offering.
+
+# membership
+memberSince: 2022-09-22
+memberType: Vendor
```
[the remark this answers is not in the document]
Comment (tomgron) [reply data/members/calm.yaml]: Change applied

File: data/members/calm.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  instagram: 
+  linkedin: https://www.linkedin.com/company/calm-oy
+  youtube:
+
+# Here you can list those who represent or have represented the member.
+#
+# Those wo no longer represent an organization might have written blog posts,
+# to keep the attribution a from/till data range can be included.
+representatives:
+   - name: Tommi Grönlund
```
[the remark this answers is not in the document]
Comment (tomgron) [reply data/members/calm.yaml]: Added picture

File: data/members/calm.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+     role: CEO
+     social:
+       twitter: 
+       linkedin: https://twitter.com/tomgron
+     description: Managing director of CALM Oy
```
[the remark this answers is not in the document]
Comment (tomgron) [reply data/members/calm.yaml]: Added longer description

File: content/members/calm.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+date: 2022-09-22T17:25:32Z
+draft: false
+id: calm
+title: CALM
+
+---
+
+## Certificate Lifecycle Management
+
+CALM Oy is the provider of https://clm.fi which is a Certificate Lifecycle Management as a Service.
```
[the remark this answers is not in the document]
Comment (tomgron) [reply content/members/calm.md]: Added a bit longer description. At the moment we are still in process of developing our offering further so I'd like to update this periodically as our offering extends (at the moment working on ACME renewals)

**Pull request merged**